features/step_definitions/authorization.js

```diff
@@ -34,6 +34,10 @@ defineSupportCode(({ Before, Given, Then }) => {
             case 'calvin':
                 Assert.strictEqual(decodedToken.username, this.username);
                 break;
+            case 'github:calvin':
+                Assert.strictEqual(decodedToken.username, this.username);
+                Assert.strictEqual(decodedToken.scmContext, this.scmContext);
```

Comment:
So we will add the `scmContext` to be part of the jwt token profile later?
https://github.com/screwdriver-cd/screwdriver/blob/master/plugins/auth/index.js#L57